AlphaGo/training/supervised_policy_trainer.py

```diff
@@ -85,6 +85,57 @@ def on_epoch_end(self, epoch, logs={}):
 }
 
 
+def load_indices_from_file(symmetries, shuffle_file):
+    # load indices from file
+    with open(shuffle_file, "r") as f:
+        indices = np.load(f)
+
+    return indices
+
+
+def set_indices(args, n_total_data_size, symmetries, shuffle_file_train, shuffle_file_val):
+
+    # create array with all position, symmetry combinations
+    shuffle_indices = np.empty(shape=[n_total_data_size * len(symmetries), 2], dtype=int)
+    for x in range(n_total_data_size):
+        for y in range(len(symmetries)):
+            shuffle_indices[x * len(symmetries) + y][0] = x
+            shuffle_indices[x * len(symmetries) + y][1] = y
+
+    # shuffle array
```

Comment:
I'd clarify in the comment briefly that this shuffles rows without affecting `x,y` pairs